Source/Scene/CreditDisplay.js

```diff
@@ -222,6 +212,97 @@ define([
         }
     }
 
+    function addStyle(selector, styles) {
+        var style = selector + ' {';
+        for (var attribute in styles) {
+            if (styles.hasOwnProperty(attribute)) {
+                style += attribute + ': ' + styles[attribute] + '; ';
+            }
+        }
+        style += ' }\n';
+        return style;
+    }
+
+    function appendCss() {
+        var head = document.head;
+        var css = document.createElement('style');
+        var style = '';
+        style += addStyle('.cesium-credit-lightbox-overlay', {
+            display : 'none',
+            'z-index' : '1', //must be at least 1 to draw over top other Cesium widgets
+            position : 'absolute',
+            top : '0',
+            left : '0',
+            width : '100%',
+            height : '100%',
+            'background-color' : 'rgba(80, 80, 80, 0.8)'
+        });
+
+        style += addStyle('.cesium-credit-lightbox', {
+            'background-color' : '#303336',
+            color : textColor,
+            position : 'relative',
+            'min-height' : lightboxHeight + 'px',
+            margin : 'auto'
+        });
+
+        style += addStyle('.cesium-credit-lightbox.cesium-credit-lightbox-expanded', {
+            border : '1px solid #444',
+            'border-radius' : '5px',
+            'max-width' : '370px'
+        });
+
+        style += addStyle('.cesium-credit-lightbox.cesium-credit-lightbox-mobile', {
+            height : '100%',
+            width : '100%'
+        });
+
+        style += addStyle('.cesium-credit-lightbox-title', {
+            padding : '20px 20px 0 20px'
+        });
+
+        style += addStyle('.cesium-credit-lightbox-close', {
+            'font-size' : '18pt',
+            cursor : 'pointer',
+            position : 'absolute',
+            top : '0',
+            right : '6px',
+            color : textColor
+        });
+
+        style += addStyle('.cesium-credit-lightbox-close:hover', {
+            color : highlightColor
+        });
+
+        style += addStyle('.cesium-credit-lightbox > ul', {
+            margin : '0',
+            padding : '12px 20px 12px 40px',
+            'font-size' : '13px'
+        });
+
+        style += addStyle('.cesium-credit-lightbox > ul > li', {
+            'padding-bottom' : '6px'
+        });
+
+        style += addStyle('.cesium-credit-expand-link', {
+            'padding-left' : '5px',
+            cursor : 'pointer',
+            'text-decoration' : 'underline',
+            color: textColor
+        });
+        style += addStyle('.cesium-credit-expand-link:hover', {
+            'color' : highlightColor
+        });
+
+        style += addStyle('.cesium-credit-text', {
+            color: textColor
+        });
+
+        css.innerHTML = style;
+
+        head.appendChild(css);
```

Comment:
Huge +1 for taking on the injected stylesheet approach.  Can you switch this from `appendChild(css)` to `insertBefore(css, head.firstChild)` or similar?  If you append this to the end of the head, these styles override the host app's styles.